templates/partials/_share_and_donate_page.liquid

```diff
@@ -13,7 +13,7 @@
       <div class="thank-you__cta">
         <span class="two-step__question">{{ 'share.two_step.cta' | t }}</span>
         <span class="two-step__declined hidden-closed">
-          {% include 'Fundraiser', ref: 'post-action-donate', freestanding: true %}
+          {% include 'Fundraiser', ref: 'post-action-donate', freestanding: true, title_i18n_key: 'share_and_donate.fundraiser_title' %}
```

Comment:
@NealJMD Ideally I'd like to be able to pass the translated string instead of the i18n key, something like `title: I18n.t('share_and_donate.fundraiser_title')`, but I couldn't get that working. That's why I ended up passing the key as an argument and then in the template using the `t` filter. Do you know if there is a way to get translations from a liquid tag?